### PR TITLE
Support tfserving

### PR DESCRIPTION
**Pull request by GarnettWang**
**support submit a tensorflow serving job:** 

```
arena submit tfserving    \
    --name=minist-demo    \
    --gpus=2    \
    --cpu=2    \
    --memory=8Gi    \
    --replicas=3    \
    --model-name=mnist
    --port=9000
    --image=tensorflow/tensorflow-serving:1.5.0-devel-gpu    \
    --model-path=/data
 ```

**What needs to do next：**

- support custom version  policy; (will comming soon.)
- support model storaged on custom hdfs system;
- arena list tfserving display tensorflow serving jobs;

### CODE REVIEW

File: cmd/arena/commands/root.go [hunk 1]
```diff
@@ -44,6 +44,7 @@ func NewCommand() *cobra.Command {
 	command.PersistentFlags().StringVar(&arenaNamespace, "arenaNamespace", "arena-system", "The namespace of arena system service, like TFJob")
 
 	command.AddCommand(NewSubmitCommand())
+	command.AddCommand(NewSubmitTFServingJobCommand())
```
Comment (cheyang) [inline cmd/arena/commands/root.go]: `NewSubmitTFServingJobCommand` is the sub command of `submit`.

**Comment by cheyang** [comment]
And do we support other serving type except TensorFlow Serving? Just like seldon and Flask?

**Comment by GarnettWang** [comment]
@cheyang，I will add the docs later.

I will add another issues/PR for seldon and Flask type. This PR just for tensorflow serving.

**Comment by wsxiaozhang** [comment]
@GarnettWang suggest to rename the command as "arena serve tf ..."

**Pull request closed without merging**